doc/modules/cross_validation.rst

```diff
@@ -467,44 +378,181 @@ Here is a usage example::
 validation that allows a finer control on the number of iterations and
 the proportion of samples on each side of the train / test split.
 
+Cross-validation iterators with stratification based on class labels.
+=====================================================================
+
+In some datasets the number of samples per class varies based on the class
+label. They are referred to commonly as datasets with imbalance in classes.
```

Comment:
``imbalanced classes``. Maybe clarify that this only applies to classification? ``In some classification datasets`` ?